packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -22,12 +24,18 @@ const MiniCart = props => {
         updateItemInCart
     } = props;
 
+    // Hooks.
+    const { innerHeight: viewportHeight } = useWindowSize();
+
     // Members.
     const classes = mergeClasses(defaultClasses, props.classes);
     const rootClass = isOpen ? classes.root_open : classes.root;
+    const rootStyle = {
+        '--minicart-height-unit': `${viewportHeight * 0.01}px`
```

Comment:
Should we debounce this?

>⚠️ Updating the value of --vh will trigger a repaint of the page and the user may experience a jump as a result. Because of this, I’m not advising that this trick should be used for every project or to replace all usage of the vh unit but only when you may need your users to have an exact viewport unit value.

Reply:
Actually we can't -- @jimbo informed me that you must have the same number of hook calls per render. Looks like this is off your plate :D